assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
-                "8R" : ["_R26L-1DME", "SENOY", "CIMBA", "FORKK", "VVMAX"],
-                "12" : ["_R30-1DME", "SENOY", "CIMBA", "FORKK", "VVMAX"],
-                "9"  : ["_R27-1DME", "SENOY", "CIMBA", "FORKK", "VVMAX"],
-                "26L": ["_R8R-1DME", "METTS", "BRBRA", "VVMAX"],
-                "26R": ["_R8L-1DME", "METTS", "BRBRA", "VVMAX"],
-                "30" : ["_R12-1DME", "METTS", "BRBRA", "VVMAX"],
-                "27" : ["_R9-1DME", "METTS", "BRBRA", "VVMAX"]
+                "KMIA8L": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA8R": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA9": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA12": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA26L": ["METTS", "BRBRA"],
+                "KMIA26R": ["METTS", "BRBRA"],
+                "KMIA27": ["METTS", "BRBRA"],
+                "KMIA30": ["METTS", "BRBRA"]
             },
+            "body": ["VVMAX"],
             "exitPoints": {
                 "HEDLY": ["HEDLY"]
             },
-            "draw": [["SENOY", "CIMBA", "FORKK", "VVMAX"], ["METTS", "BRBRA", "VVMAX", "HEDLY*"]]
+            "draw": [
+                ["SENOY", "CIMBA", "FORKK", "VVMAX", "HEDLY*"],
+                ["METTS", "BRBRA", "VVMAX"]
+            ]
+        },
+        "HITAG2": {
+            "icao": "HITAG2",
+            "name": "Hi-tag Two",
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": []
+            },
+            "body": ["POTTR", "HITAG"],
+            "exitPoints": {
+                "HEDLY": ["KAINS", "FORKK", "HOSFU", "VVMAX", "HEDLY"],
+                "PADUS": ["SHALW", "PADUS"],
+                "VALLY": ["SHALW", "VALLY"],
+                "WINCO": ["KAINS", "FORKK", "HOSFU", "BORLE", "WINCO"]
+            },
+            "draw": [
+                ["POTTR", "HITAG"],
+                ["HITAG", "KAINS", "FORKK", "HOSFU"],
+                ["HOSFU", "VVMAX", "HEDLY*"],
+                ["HOSFU", "BORLE", "WINCO*"],
+                ["HITAG", "SHALW"],
+                ["SHALW", "PADUS*"],
+                ["SHALW", "VALLY*"]
+            ]
+        },
+        "JONZI2": {
+            "icao": "JONZI2",
+            "name": "Joanzie Two",
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": ["_RWY30JONZI091"]
+            },
+            "body": ["JONZI", "VKZ"],
+            "exitPoints": {
+                "EONNS": ["LUVLY", "MANNN", "EONNS"],
+                "MNATE": ["NREQZ", "MNATE"],
+                "SKIPS": ["SKIPS"]
+            },
+            "draw": [
+                ["JONZI", "VKZ"],
+                ["VKZ", "LUVLY", "MANNN", "EONNS*"],
+                ["VKZ", "NREQZ", "MNATE*"],
+                ["VKZ", "SKIPS*"]
+            ]
+        },
+        "MIA5": {
```

Comment:
MIA5 is now MIA6

Reply:
reverified and updated all SIDs/STARs.

done